enterprise/server/cmd/server/main.go

```diff
@@ -237,6 +239,13 @@ func main() {
 			log.Fatalf("Error configuring scheduler server: %v", err)
 		}
 		realEnv.SetSchedulerService(schedulerServer)
+
+		// Fulfill internal remote execution requests locally.
+		conn, err := grpc_client.DialTarget(fmt.Sprintf("grpc://localhost:%d", *libmain.GRPCPort))
+		if err != nil {
+			log.Fatalf("Error initializing remote execution client: %s", err)
+		}
+		realEnv.SetRemoteExecutionClient(repb.NewExecutionClient(conn))
 	}
```

Comment:
does this work correctly if remote execution is not configured?

Reply:
It's hard to tell on GitHub but this code is within a conditional that checks whether remote execution is configured. For good measure I also added a nil check on executionClient within `waitForWorkflowInvocationCreated` and return UnimplementedError if so.